docs/index.bs

```diff
@@ -1453,7 +1453,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
         1. Run these substeps <a>in parallel</a>:
             1. Let |newContext| be a new [=top-level browsing context=].
             1. [=Queue a task=] to run the following steps on |newContext|'s {{Window}} object's [=environment settings object=]'s [=responsible event loop=] using the [=user interaction task source=]:
-                1. *HandleNavigate*: [=Navigate=] |newContext| to |url| with [=exceptionsEnabled=] true, and [=historyHandling=] "<code>replace</code>".
+                1. *HandleNavigate*: [=Navigate=] |newContext| to |url| with [=exceptionsEnabled=] true, and <i>[=navigate/historyHandling=]</i> "<code>replace</code>".
```

Comment:
I think you will need to change this to `navigate/exceptionsEnabled` after https://github.com/whatwg/html/pull/10308 lands.

Reply:
done.